docs/users-guide/10-metabot.md

```diff
@@ -0,0 +1,44 @@
+# Getting answers in Slack with Metabot
+
+You can already send data to Slack on  a set schedule via [Pulses](http://www.metabase.com/docs/latest/users-guide/07-pulses) but what about when you need an answer right now?. Say hello to Metabot.
+
+Metabot helps add context to conversations you’re having in Slack by letting you see results from Metabase as part of the conversation.
+
+## Connecting to Slack.
+To use Metabot with Slack you’ll first need to connect Metabase to your Slack with an API token.
+
+See [Setting up Slack](http://www.metabase.com/docs/v0.15.1/administration-guide/07-setting-up-slack) for more information.
+
+
+## What can Metabot do?
+
+Metabot can show individual questions and also lists of questions that have already been asked in Metabase.
+
+If you ever need help remembering what Metabot can do, just type ```metabot help``` in Slack.
+
+![Metabot help](images/metabot/MetabotHelp.png)
+
+## Showing questions
+
+To see a question from Metabase in Slack type
+“metabot show” and then specify the name of a saved question. If you have several similarly named questions Metabot will ask you to differentiate between the two by typing the number next to the name.
```

Comment:
Yeah if you don't put quotes around the string metabot will basically just grab the first word and go off of that. So it's not "required" necessarily but it does change the results quite a bit.